### PR TITLE
IDL docs.

### PR DESCRIPTION
**Pull request by bkaradzic**

### CODE REVIEW

**Comment by bkaradzic** [comment]
There is problem with defaults that are defines: `{ default = BGFX_BUFFER_NONE }`

**Comment by cloudwu** [comment]
I suggest `default = "BGFX_BUFFER_NONE" ` instead of `default = BGFX_BUFFER_NONE`, because it's a specific type of bgfx. ( idl.lua is  independent to bgfx now)

If we defines macros in IDL later, we can use something like `BGFX.BUFFER_NONE`.

File: bgfx.idl [hunk 1]
```diff
@@ -631,6 +647,7 @@ func.weldVertices
 	.data    "const void*"        --- Vertex stream.
 	.num     "uint16_t"           --- Number of vertices in vertex stream.
 	.epsilon "float"              --- Error tolerance for vertex position comparison.
+--	 { default = 0.001f }
```
Comment (cloudwu) [inline bgfx.idl]: You may use `default = 0.0001` here, or `default = "0.001f"` }

File: bgfx.idl [hunk 2]
```diff
@@ -438,6 +438,19 @@ struct.Attachment
 	.layer   "uint16_t"      --- Cubemap side or depth layer/slice.
 	.resolve "uint8_t"       --- Resolve flags. See: `BGFX_RESOLVE_*`
 
+--- Init attachment.
+func.Attachment.init
+	"void"
+	.handle "TextureHandle" --- Render target texture handle.
+	.access "Access::Enum"  --- Access. See `Access::Enum`.
+--	 { default = Access::Write }
```
Comment (cloudwu) [inline bgfx.idl]: `default = "Access::Write" `

Comment (bkaradzic) [reply bgfx.idl]: Ah ok, I was basing it on `default = INT32_MAX` that doesn't have quotes.

**Pull request merged**